class.jetpack-import-stats.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+class Jetpack_Import_Stats {
+	static $known_importers = array(
```

Comment:
It would be nice to have docblock comments for the properties and methods in this class to quickly grok what key/value maps are (like classes to "friendly" names) and types for params.

Reply:
Should these class properties be `private`?